### PR TITLE
fix(example): point picking, wrong point's property in console.log

### PR DESCRIPTION
**Pull request by gchoqueux**
Point picking, wrong point's property in console.log

### CODE REVIEW

File: examples/pointcloud.html [hunk 1]
```diff
@@ -197,7 +197,11 @@ <h3>Available pointclouds:</h3>
                         var pick = view.pickObjectsAt(event, 5, pointcloud);
 
                         for (const p of pick) {
-                            console.log('Selected point #' + p.index + ' in Points "' + p.object.owner.name + '"');
+                            console.info('Selected point #' + p.index + ' in position ("' +
+                                p.object.position.x + ',' +
+                                p.object.position.y + ',' +
+                                p.object.position.y + ',' +
```
Comment (zarov) [inline examples/pointcloud.html]: It is `z` and you can remove the comma at the end.

File: examples/pointcloud.html [hunk 2]
```diff
@@ -197,7 +197,11 @@ <h3>Available pointclouds:</h3>
                         var pick = view.pickObjectsAt(event, 5, pointcloud);
 
                         for (const p of pick) {
-                            console.log('Selected point #' + p.index + ' in Points "' + p.object.owner.name + '"');
+                            console.info('Selected point #' + p.index + ' in position ("' +
+                                p.object.position.x + ',' +
```
Comment (zarov) [inline examples/pointcloud.html]: Add a space after the comma, same for others.

File: examples/pointcloud.html [hunk 3]
```diff
@@ -197,7 +197,11 @@ <h3>Available pointclouds:</h3>
                         var pick = view.pickObjectsAt(event, 5, pointcloud);
 
                         for (const p of pick) {
-                            console.log('Selected point #' + p.index + ' in Points "' + p.object.owner.name + '"');
+                            console.info('Selected point #' + p.index + ' in position ("' +
```
Comment (zarov) [inline examples/pointcloud.html]: Remove the last `"` or add it at the end.

**Review by zarov — COMMENTED** [review]
LGTM, some changes necessary

**Comment by gchoqueux** [comment]
@zarov updated

**Review by zarov — APPROVED** [review]

**Pull request merged**